src/traces/scatter/index.js

```diff
@@ -9,34 +9,31 @@
 
 'use strict';
 
-var d3 = require('d3');
-var isNumeric = require('fast-isnumeric');
-
-var Lib = require('../../lib');
-
-var Fx = require('../../plots/cartesian/graph_interact');
-var Axes = require('../../plots/cartesian/axes');
-
-var Color = require('../../components/color');
-var Colorscale = require('../../components/colorscale');
-var Drawing = require('../../components/drawing');
-var ErrorBars = require('../../components/errorbars');
+var Scatter = {};
 
 var subtypes = require('./subtypes');
+Scatter.hasLines = subtypes.hasLines;
+Scatter.hasMarkers = subtypes.hasMarkers;
+Scatter.hasText = subtypes.hasText;
+Scatter.isBubble = subtypes.isBubble;
 
-var scatter = module.exports = {};
-
-scatter.hasLines = subtypes.hasLines;
-scatter.hasMarkers = subtypes.hasMarkers;
-scatter.hasText = subtypes.hasText;
-scatter.isBubble = subtypes.isBubble;
-
-scatter.selectPoints = require('./select');
-
-scatter.moduleType = 'trace';
-scatter.name = 'scatter';
-scatter.categories = ['cartesian', 'symbols', 'markerColorscale', 'errorBarsOK', 'showLegend'];
-scatter.meta = {
+// traces with < this many points are by default shown
+// with points and lines, > just get lines
+Scatter.attributes = require('./attributes');
+Scatter.supplyDefaults = require('./defaults');
+Scatter.cleanData = require('./clean_data');
+Scatter.calc = require('./calc');
+Scatter.arraysToCalcdata = require('./arrays_to_calcdata');
+Scatter.plot = require('./plot');
+Scatter.colorbar = require('./colorbar');
+Scatter.style = require('./style');
+Scatter.hoverPoints = require('./hover');
+Scatter.selectPoints = require('./select');
+
+Scatter.moduleType = 'trace';
+Scatter.name = 'scatter';
+Scatter.categories = ['cartesian', 'symbols', 'markerColorscale', 'errorBarsOK', 'showLegend'];
+Scatter.meta = {
```

Comment:
@mdtusz I tried to keep the same style as the other trace module index files. I hope you like it.